flux-cli/src/main/java/com/marklogic/flux/impl/importdata/SplitterParams.java

```diff
@@ -183,7 +183,7 @@ public Map<String, String> makeOptions() {
             Options.WRITE_SPLITTER_SIDECAR_URI_SUFFIX, uriSuffix,
             Options.WRITE_CLASSIFIER_HOST, classifierHost,
             Options.WRITE_CLASSIFIER_PORT, classifierPort,
-            Options.WRITE_CLASSIFIER_PROTOCOL, classifierProtocol,
+            Options.WRITE_CLASSIFIER_PROTOCOL, classifierProtocol ? "https" : (classifierHost != null) ? "http" : null,
```

Comment:
I think it's worth changing the Spark connector option as well to be boolean. That way, the user doesn't need to know that the valid values are "https" and "http". And if the user doesn't set anything, and the classifier is host is set, it should default to "http". 

Then you can just do `classifierHttps ? "true" : null`. (I would rename it to `classifierHttps` as well since the value is no longer a protocol).